packages/ui-components/src/error-pages/page-401/__snapshots__/Page401.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Error Page 401 tests Test LoadingPage renders 1`] = `[Function]`;
```

Comment:
Why this snapshot? Maybe svg is not handled by jsdom?

In this case, this snapshot is not useful, there is another way to test this? If not, we could return to the expect of existence.

Reply:
No, I just didn't update the snapshots after the fix to the tests! Now it should make sense and be useful